src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10991,15 +10991,15 @@ void WriteVeriSumTable(EnergyPlusData &state)
                     curArea += frameArea;
                 }
                 int const zonePt = thisSurf.Zone;
-                auto const &thisZone = state.dataHeatBal->Zone(zonePt);
 
-                bool const isConditioned = (zonePt > 0) && (thisZone.SystemZoneNodeNumber > 0);
+                bool const isConditioned = (zonePt > 0) && (state.dataHeatBal->Zone(zonePt).SystemZoneNodeNumber > 0);
```

Comment:
This boolean short circuit works fine as long as you defer array lookup until you know it is valid.

Reply:
Same type of CppCheck change error I made in HVACManager.